spec/index.bs

```diff
@@ -883,6 +883,22 @@ An [=adapter=] has the following internal slots:
 
         Each adapter limit must be the same or [=limit/better=] than its default value
         in [=supported limits=].
+
+    : <dfn>\[[current]]</dfn>, of type boolean
+    ::
+        Indicates whether the adapter is allowed to vend new devices at this time.
+        Its value may change at any time.
```

Comment:
I changed this from "only outside of user tasks" to "at any time" because this state is not part of the content process, so changing it between user tasks doesn't make sense.